### ISSUE TITLE
Defensive abilities named Resist ____ cause issues.

### ISSUE THREAD

**Issue by baileymh** [Lavaeolous/PF1-StatBlock-Converter-Module#467]
```
Jeslyn
CR 6
XP 2,400
Female half-elf oracle 7
CE Medium humanoid (elf, human)
Init +1; Senses low-light vision; Perception +11

DEFENSE
AC 22, touch 15, flat-footed 20 (+6 armor, +3 deflection, +1 Dex, +1 dodge, +1 natural)
hp 80 (7d8+45)
Fort +8, Ref +5, Will +6; +4 vs. disease, +2 vs. enchantments
Defensive Abilities resist life; Immune disease, sickness, sleep
Weaknesses sickened

OFFENSE
Spd 30 ft.
Melee +1 scythe +9 (2d4+5/x4)
Space 5 ft.; Reach 5 ft.
Special Attacks soul siphon 1/day

Oracle Spells Known (CL 7th; concentration +11)
3rd (5/day)—animate dead, bestow curse (DC 17), inflict serious wounds (DC 17), summon monster III
2nd (7/day)—bull's strength, bear's endurance, false life, hold person (DC 16), inflict moderate wounds (DC 16)
1st (7/day)—cause fear (DC 15), command (DC 15), divine favor, inflict light wounds (DC 15), obscuring mist, sanctuary (DC 15), shield of faith
0 (at will)—bleed (DC 14), detect magic, detect poison, guidance, read magic, resistance, stabilize
Mystery bones

TACTICS
Before Combat Jeslyn casts bull's strength, bear's endurance, false life, and shield of faith once she fears the PCs are drawing near.
During Combat Jeslyn casts divine favor on the first round of combat, then attacks with her scythe. If she's having trouble hitting foes, she abandons the scythe and instead uses her inflict spells. She uses bestow curse to lower ability scores against foes-she's particularly fond of reducing Wisdom scores for clerics, but doesn't bother using spells with saving throws against paladins.
Morale Jeslyn fights to the death.

STATISTICS
Str 16, Dex 13, Con 18, Int 10, Wis 8, Cha 18
Base Atk +5; CMB +8; CMD 23
Feats Craft Magic Arms and Armor, Craft Wondrous Item, Dodge, Martial Weapon Proficiency (scythe), Skill Focus (Perception)
Skills Intimidate +13, Knowledge (planes) +9, Knowledge (religion) +9, Linguistics +3, Perception +11
Languages Abyssal, Common, Elven, Hallit, Necril
SQ oracle's curse (wasting), revelations (armor of bones, resist life, soul siphon)
Combat Gear scroll of dispel magic (3); Other Gear +1 scythe, amulet of natural armor +1, cloak of resistance +2, keys to areas P15 and P18, 350 gp in onyx gems for animate dead, 32 gp
```

Results in 
![image](https://user-images.githubusercontent.com/48079051/122660292-6493e380-d14e-11eb-9e80-d596066e8a11.png)

Rather than when the name was changed: 
![image](https://user-images.githubusercontent.com/48079051/122660297-6d84b500-d14e-11eb-82bb-869da6759b80.png)

Not sure if/how much you can do about this.

**Comment by Lavaeolous** [comment]
Yeah, i can fix that by ignoring "resist" (the keyword for resistances) in the defensive abilities line. Fix will be included in 3.3.1

**Comment by Lavaeolous** [comment]
Fixed in 3.3.1
